css/css-typed-om/the-stylepropertymap/properties/logical.html

```diff
@@ -13,15 +13,59 @@
 <script>
 'use strict';
 
+const logical = {
+  sides: [
+    'block-start', 'block-end', 
+    'inline-start', 'inline-end',
+  ],
+  shorthands: [
+    'block', 
+    'inline',
+  ],
+};
+
 for (const prefix of ['margin-', 'inset-', 'padding-']) {
-  for (const suffix of ['block-start', 'block-end', 'inline-start', 'inline-end']) {
-    runPropertyTests(prefix + suffix, [
+  for (const side of logical.sides) {
```

Comment:
Nit: why not `logical.shorthands`?

Reply:
nice, thanks for catching the refactor regression 👍